src/test/pages/admin/userManagement/userPage.ts

```diff
@@ -29,11 +116,201 @@ export default class AdminMenuPage {
         await this.passWord.fill("admin123");
         await this.loginBtn.click();
     }
-    async createUser() {
+    async accessUserPage() {
+        await this.adminMenu.click();
+        await this.userManagement.click();
+    }
+    async afterAccessUserPage() {
+        await expect(this.titlePage).toBeVisible();
+        await expect(this.usernameLabel).toBeVisible();
+        await expect(this.usernameFieldSearch).toBeEditable();
+        await expect(this.userRoleLabel).toBeVisible();
+        await expect(this.userRole).toBeVisible();
+        await expect(this.employeeNameLabel).toBeVisible();
+        await expect(this.employeeName).toBeEditable()
+        await expect(this.statusLabel).toBeVisible();
+        await expect(this.status).toBeVisible();
+        await expect(this.resetBtn).toBeVisible();
+        await expect(this.searchBtn).toBeVisible();
+        await expect(this.addUserBtn).toBeVisible();
+        await expect(this.checkBox).toBeVisible();
+        await expect(this.usernameColumn).toBeVisible();
+        await expect(this.userRoleColumn).toBeVisible();
+        await expect(this.employeeNameColumn).toBeVisible();
+        await expect(this.statusColumn).toBeVisible();
+        await expect(this.actionColumn).toBeVisible();
+    }
+    async createEssUser() {
+        //employee: string, username: string, pass: string, confirm: string
         await this.adminMenu.click();
         await this.addBtn.click();
         await this.userRole.click();
+        await this.userRoleOption.click();
+        await this.status.click();
+        await this.statusOption.click();
+        await this.employeeName.fill("t");
+        await this.employeeOption.click();
+        await this.usernameField.fill("usernamenttheu");
+        await this.passwordField.fill("admin123");
+        await this.confirmPassword.fill("admin123");
+        await this.submitBtn.click();
+    }
+    async afterCreateEssUser() {
+        await this.successToast.waitFor({ state: 'visible', timeout: 10000 });
+        await expect(this.newEssUser).toBeVisible();
+    }
+    async createAdminUser() {
+        //employee: string, username: string, pass: string, confirm: string
+        await this.adminMenu.click();
+        await this.addBtn.click();
+        await this.userRole.click();
+        await this.adminRoleOption.click();
+        await this.status.click();
+        await this.statusOption.click();
+        await this.employeeName.fill("t");
+        await this.employeeOption.click();
+        await this.usernameField.fill("usernamenttheuAdmin");
```

Comment:
This would be better if you put these value as parameters instead of hard coding:

```
async createAdminUser(username: string, password: string): {
    await this.usernameField.fill(username);
    await this.passwordField.fill(password);
    await this.confirmPassword.fill(password);
}
```